deploy/examples/csi-operator-manifests/csi_v1alpha1_config.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: csi.ceph.io/v1alpha1
```

Comment:
@subhamkrai IMO we can automate to pull CRD and operator deployment (still need to customize it) RBAC's only nothing else

Reply:
so we don't want to pull default examples?

Reply:
Yes correct

Reply:
@Madhu-1 @travisn should I create a sub-folder to have all crds, rbac files in individual sub-folder. Example `deploy/examples/csi-operator/crds` and `deploy/examples/csi-operator/rbac`?

Reply:
I'd suggest keep a single folder. In the other comment, I suggested perhaps to combine the files.